website/static/js/newAndNoteworthyPlugin.js

```diff
@@ -73,6 +79,10 @@ var NewAndNoteworthy = {
         };
     },
     view : function(ctrl) {
+        if (ctrl.errorLoading) {
```

Comment:
I can change it to a m.prop? I wasn't expecting the error to resolve but maybe there's a scenario where it would.